setup-qemu-kernel.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash 
+
+
+set -ex 
+
+
+cd "$(dirname "$0")"
+
+
+ARCH=arm64
+ROOT_DIR=$HOME 
+NUM_CORES=${NUM_CORES:-$(nproc)}
+IMAGE_FILE=""
+KERNEL_VERSION="linux-6.1.34"
```

Comment:
Why this kernel version?  What does it correspond to?  Is that the one that we're building for.  Should comment this choice.